### PR TITLE
[KCC] Support Helm KMS keyring solution

### PR DESCRIPTION
**Pull request by harshatech**
Config Connector compatible YAML files to create a KMS key ring in your desired project, and grant a specific member a role (default to roles/cloudkms.admin) for accessing the kMS key ring that just created

### CODE REVIEW

File: config-connector/solutions/iam/helm/kms-key-ring/README.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    helm install . --dry-run --set iamPolicyMember.iamMember=user:name@example.com --generate-name
+
+    # install your chart
+    helm install . --set iamPolicyMember.iamMember=user:name@example.com --generate-name
+    ```
+
+1. _Optionaly_, you can set the name of the KMS keyring (defaults to `allowed-ring`), set the location of the ring (defaults to `us-central1`) and the role to grant (defaults to `roles/pubsub.editor`, full list of roles [here](https://cloud.google.com/iam/docs/understanding-roles#cloud-kms-roles)) by explictly setting them when installing the solution:
+
+    ```bash
+    # install your chart with a difirent name of the KMS keyring and location
```
Comment (AlexBulankou) [inline config-connector/solutions/iam/helm/kms-key-ring/README.md:60]: typo, different

File: config-connector/solutions/iam/helm/kms-key-ring/templates/iampolicymember.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: iam.cnrm.cloud.google.com/v1beta1
+kind: IAMPolicyMember
+metadata:
+  name: ring-iam-member
+spec:
+  member: {{ required "IAM member is required!" .Values.iamPolicyMember.iamMember }}
+  role: {{ .Values.iamPolicyMember.role }}
```
Comment (AlexBulankou) [inline config-connector/solutions/iam/helm/kms-key-ring/templates/iampolicymember.yaml:20]: role and name are also required, can you please add?

File: config-connector/solutions/iam/helm/kms-key-ring/templates/kmskeyring.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: kms.cnrm.cloud.google.com/v1beta1
+kind: KMSKeyRing
+metadata:
+  name: {{ .Values.KMSKeyRing.name }}
```
Comment (AlexBulankou) [inline config-connector/solutions/iam/helm/kms-key-ring/templates/kmskeyring.yaml:17]: name and location are also required, can you please add?

Comment (maqiuyujoyce) [reply config-connector/solutions/iam/helm/kms-key-ring/templates/kmskeyring.yaml:17]: @AlexBulankou In our previous helm solutions, only the fields that have PLACEHOLDER values in the corresponding kpt solutions are marked as required. Do we want to change that strategy?

Comment (AlexBulankou) [reply config-connector/solutions/iam/helm/kms-key-ring/templates/kmskeyring.yaml:17]: Let's keep the same approach as before then.

**Pull request merged**